.gitignore

```
@@ -1,7 +1,6 @@
+_test/check_exercises_tmp/
```

```suggestion
_test/check_exercises_tmp/
_test/check_exercises
_test/check_uuids
```

One change from this PR: we no longer ignore non-exe binaries produced from `nim c _test/check_*.nim`.

I don't think that's much of a problem, since `nim r` will be supported too now (see #349), and we don't have ignore rules for binaries produced by compiling other Nim files in this repo (like an `example.nim`). But this suggestion is an alternative.

---

I almost never compile without `r`, but I'm not sure? 🤷‍♂️